FWCore/Framework/src/NumberOfConcurrentIOVs.cc

```diff
@@ -35,6 +35,13 @@ namespace edm {
       }
     }
 
+    void NumberOfConcurrentIOVs::initialize(unsigned int nStreams, unsigned int nConcurrentLumis) {
+      maxConcurrentIOVs_ = nStreams;
+      if (maxConcurrentIOVs_ > nConcurrentLumis) {
+        maxConcurrentIOVs_ = nConcurrentLumis;
```

Comment:
Should we print a message saying the max was changed?

Reply:
I added a LogInfo message when the number of streams or concurrent lumis restricts the number of concurrent IOVs below what was configured.